apps/beets-frontend-v3/lib/modules/lst/hooks/useGetStakedSonicData.tsx

```diff
@@ -1,6 +1,6 @@
 import { GetStakedSonicDataDocument } from '@repo/lib/shared/services/api/generated/graphql'
 import { useQuery } from '@apollo/client'
-import { minutesToMilliseconds } from 'date-fns'
+import { minutesToMilliseconds } from 'date-fns/minutesToMilliseconds'
```

Comment:
isn't this function available at the same place as all the others (e.g. `secondsToMilliseconds`)?

Reply:
that's what i thought too

https://vercel.com/beethovenx/mono-beets-v3/H3KTYEPEQ1m7Br291NabnVtgUy62?filter=errors

Reply:
ok reverted to `import { minutesToMilliseconds } from 'date-fns'` and now it builds?!?